src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -123,15 +125,50 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
    * EUI-3732 - Breathing space data not persisted on Previous button click with ExpUI Demo
    */
   toPreviousPage() {
+    this.validationErrors = [];
     let caseEventData: CaseEventData = this.buildCaseEventData();
     caseEventData.data = caseEventData.event_data;
     this.updateFormData(caseEventData);
     this.previous();
     CaseEditPageComponent.setFocusToTop();
   }
 
+  // Adding validation message to show it as Error Summary
+  generateErrorMessage(fields: CaseField[]) {
```

Comment:
_Nitpick:_
Should have a return type of `: void`.

Reply:
updated